### PR TITLE
Fix for tips' usages

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

File: src/main/scala/org/constellation/checkpoint/CheckpointAcceptanceService.scala [hunk 1]
```diff
@@ -216,7 +217,7 @@ class CheckpointAcceptanceService[F[_]: Concurrent: Timer](
 
         _ <- logger.debug(s"[Accept checkpoint][${cb.baseHash}] Accept data")
         _ <- checkpointService.put(checkpoint.copy(height = maybeHeight))
-        _ <- checkpointParentService.incrementChildrenCount(cb)
+        _ <- checkpointParentService.incrementChildrenCount(cb) // TODO: is it used?
```
Comment (tbekas) [inline src/main/scala/org/constellation/checkpoint/CheckpointAcceptanceService.scala:220]: What about this TODO, do you think it should get covered in this PR?

**Review by tbekas — DISMISSED** [review]
LGTM

File: src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala [hunk 2]
```diff
@@ -27,9 +27,9 @@ class SnapshotHashIntegrityTest extends AnyFreeSpec with Matchers with BeforeAnd
 
   "snapshot info read" in {
     val storage = SnapshotInfoLocalStorage[IO]("src/test/resources")
-    val snapshotInfo = storage.read("snapshot-info").rethrowT.unsafeRunSync()
+    val snapshotInfo = storage.read("snapshot-info-v2").rethrowT.unsafeRunSync()
```
Comment (TheMMaciek) [inline src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala:30]: Is that the old snapshot? If yes why did the hash change?

File: src/main/scala/org/constellation/infrastructure/endpoints/CheckpointEndpoints.scala [hunk 3]
```diff
@@ -57,14 +58,16 @@ class CheckpointEndpoints[F[_]](implicit F: Concurrent[F], C: ContextShift[F]) e
...
+    checkpointAcceptanceService: CheckpointAcceptanceService[F],
+    tipService: ConcurrentTipService[F]
   ): HttpRoutes[F] = HttpRoutes.of[F] {
     case req @ POST -> Root / "finished" / "checkpoint" =>
       for {
         finishedCheckpoint <- req.decodeJson[FinishedCheckpoint]
         baseHash = finishedCheckpoint.checkpointCacheData.checkpointBlock.baseHash
         _ <- logger.debug(s"Handle finished checkpoint for cb: $baseHash")
         _ <- metrics.incrementMetricAsync[F]("peerApiRXFinishedCheckpoint")
+        _ <- tipService.registerUsages(finishedCheckpoint)
```
Comment (TheMMaciek) [inline src/main/scala/org/constellation/infrastructure/endpoints/CheckpointEndpoints.scala:70]: I guess it's not harmful to register usages here and in accept method, as we are keeping child hashes as a set right?

File: src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala (hunk 2 above)
Comment (marcinwadon) [reply src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala:30]: No, just randomly selected snapshot from the testnet, to test the storage service

File: src/main/scala/org/constellation/infrastructure/endpoints/CheckpointEndpoints.scala (hunk 3 above)
Comment (marcinwadon) [reply src/main/scala/org/constellation/infrastructure/endpoints/CheckpointEndpoints.scala:70]: exactly, that's the point. we need to register usages here as this is the first entry point to the node. registering in acceptance is a backup

**Pull request closed without merging**